system-services/meticulous-rauc.service

```diff
@@ -4,8 +4,8 @@ ConditionKernelCommandLine=|rauc.slot
 StartLimitInterval=20000
 StartLimitBurst=5
 
-After=meticulous-backend.service meticulous-dial.service meticulous-watcher.service
```

Comment:
We could keep the dial here as it will only start if the weston has also started. We should add sd_notify to the dial as well at some point